server/market/market.go

```diff
@@ -1148,15 +1149,15 @@ func (m *Market) handlePreimageResp(msg *msgjson.Message, reqData *piData) {
 // collectPreimages solicits preimages from the owners of each of the orders in
 // the provided queue with a 'preimage' ntfn/request via AuthManager.Request,
 // and returns the preimages contained in the client responses. This function
-// can block for up to 5 seconds (piTimeout) to allow clients time to respond.
+// can block for up to 20 seconds (piTimeout) to allow clients time to respond.
 // Clients that fail to respond, or respond with invalid data (see
 // handlePreimageResp), are counted as misses.
 func (m *Market) collectPreimages(orders []order.Order) (cSum []byte, ordersRevealed []*matcher.OrderRevealed, misses []order.Order) {
 	// Compute the commitment checksum for the order queue.
 	cSum = matcher.CSum(orders)
 
 	// Request preimages from the clients.
-	piTimeout := 5 * time.Second
+	piTimeout := 20 * time.Second
```

Comment:
In retrospect, I'm sorta surprised 5 sec was enough.  Probably should switch to default (30 sec), or longer as per https://github.com/decred/dcrdex/issues/592, but we don't want to hold up the epoch queue pipeline too much.